cfme/tests/configure/test_access_control.py

```diff
@@ -125,11 +125,6 @@ def test_user_password_required_error_validation():
         user.create()
 
 
-@pytest.mark.meta(
-    blockers=[
-        BZ(1118040, unblock=lambda appliance_version: appliance_version < "5.3")
```

Comment:
Be careful with further removing the bugs with unblock= having 5.3 in the future. This one is closed already so it does not matter.